app/src/sharedTest/java/org/oppia/app/topic/practice/TopicPracticeFragmentTest.kt

```diff
@@ -160,38 +163,64 @@ class TopicPracticeFragmentTest {
   fun testTopicPracticeFragment_loadFragment_configurationChange_startButtonRemainsInactive() {
-    onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
-    activityScenario.onActivity { activity ->
-      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
-    }
-    activityScenario.recreate()
-    onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
-    onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(not(isClickable())))    }
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicPracticeFragment_loadFragment_selectSkills_configurationChange_startButtonRemainsActive() {
-    onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).perform(click())
-    activityScenario.onActivity { activity ->
-      activity.requestedOrientation = Configuration.ORIENTATION_LANDSCAPE
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(atPositionOnView(R.id.topic_practice_skill_list, 1, R.id.subtopic_check_box)).perform(click())
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.topic_practice_skill_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(10))
+      onView(withId(R.id.topic_practice_start_button)).check(matches(isClickable()))    }
+  }
+
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeorientation_titleiscorrect() {
+    launchTopicActivityIntent(FRACTIONS_TOPIC_ID).use {
+      onView(
+        allOf(
+          withText(TopicTab.getTabForPosition(2).name),
+          isDescendantOfA(withId(R.id.topic_tabs_container))
+        )
+      ).perform(click())
+      onView(withId(R.id.master_skills_text_view)).check(matches(withText(R.string.topic_practice_master_these_skills)))
+      onView(isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.master_skills_text_view)).check(matches(withText(R.string.topic_practice_master_these_skills)))
```

Comment:
Press `Control + Options + L` to fix indentation. Also, this file contains some unused imports.

Reply:
When I do that, there are many functions that become multi-line but were single-line previously. Should I proceed with Control+Options+L ?

Reply:
Yes do proceed with it, if there are some unnecessary changes introduced I will check that in another review.

Reply:
Done PTAL.